code/client/munkilib/appleupdates.py

```diff
@@ -640,7 +640,7 @@ def _GetPreferredLocalization(self, list_of_localizations):
         localizations. Returns a single language/localization name.'''
         # pylint: disable=no-self-use
         localization_preferences = (
-            munkicommon.pref('AppleSoftwareUpdateLanguages') or ['English'])
+            munkicommon.getAppleLanguages() or ['English'])
```

Comment:
hmm... the problem is, we have multiple users with different languages on one mac...